source/hassio/HassOS-FAQ.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: "Home Assistant OS FAQ"
+description: "Frequently Asked Questions for HA OS"
+---
+
+# Home Assistant OS FAQ
+
+## Is USB Boot for the Pi4 supported? Is the Pi4 with 8GB RAM supported?
+
+The bootloader for Home Assistant OS (uboot) does not yet support Pi4 booting from USB. Support is likely a month or two away as of July 2020.
+
+The Pi4 which comes with 8GB RAM is similarly waiting for support from upstream.
+
```

Comment:
This was a bit of a confusion sentence, as it might imply all Pi4's had 8GB RAM.

```suggestion
The Pi4, with 8GB RAM, is similarly waiting for support from upstream.
```